doc/source/examples/multiphysics/laser-heating/laser-heating.rst

```diff
@@ -0,0 +1,231 @@
+Features
+----------------------------------
+- Solver: ``gls_navier_stokes_3d`` 
+- Laser heat source
+- Convection-radiation heat transfer boundary condition
+- Unsteady problem handled by an adaptive BDF1 time-stepping scheme 
+- Time-dependent laser path
+- Mesh adaptation using temperature
+
+
+------------------------
+Location of the example
+------------------------
+``examples/multiphysics/laser_heating/laser_heating.prm``
+
+
+-----------------------------
+Description of the case
+-----------------------------
+
+A laser beram heats a three-dimensional solid block. The laser beam is emitted perpendicular to the top surface of the block in negative z direction. The laser path changes with time. The laser beam radius and penetration depth are 0.00005 m and 0.00005 m. Because of this small radius and penetration depth, we use apaptive mesh refinement based on the temperature. Thermal boundary conditions are ``convection-radiation`` with a convective heat transfer coefficient of 5 and an emissivity of 0.4. The corresponding parameter file is 
+``laser_heating.prm``.
+
+--------------
+Parameter file
+--------------
+
+Time integration is handled by a 2nd order backward differentiation scheme `(bdf1)` (for a better temporal accuracy), for a :math:`0.003` seconds simulation time with a constant
+time step of :math:`5.0 \times 10^{-5}` seconds.
+
+
+.. code-block:: text
+
+    # --------------------------------------------------
+    # Simulation Control
+    #---------------------------------------------------
+    subsection simulation control
+        set method                      = bdf1
+        set time end                    = 0.003
+        set time step                   = 0.00005
+        set output name                 = laser_heating
+        set output frequency            = 1
+        set output path                 = ./output/
+        set subdivision                 = 1
+    end
+
+
+All the four boundary conditions are ``noslip``, and the heat transfer boundary conditions are ``convection-radiation``.
+
+.. code-block:: text
+
+    # --------------------------------------------------
+    # Boundary Conditions
+    #---------------------------------------------------
+    subsection boundary conditions
+      set number                  = 6
+        subsection bc 0
+            set id = 0
+            set type              = noslip
+        end
+        subsection bc 1
+            set id = 1
+            set type              = noslip
+        end
+        subsection bc 2
+            set id = 2
+            set type              = noslip
+        end
+        subsection bc 3
+            set id = 3
+            set type              = noslip
+        end
+        subsection bc 4
+            set id = 4
+            set type              = noslip
+        end
+        subsection bc 5
+            set id = 5
+            set type              = noslip
+        end
+    end
+    subsection boundary conditions heat transfer
```

Comment:
Since all boundary conditions are identical, do you need to have all of them colorized with different ids?

Reply:
Done.